wire-ios-system/Source/Logging/CocoaLumberjackLogger.swift

```diff
@@ -60,7 +60,13 @@ public class CocoaLumberjackLogger: LoggerProtocol {
     }
 
     private func log(_ message: LogConvertible, attributes: LogAttributes?, level: DDLogLevel) {
-        var entry = "\(message.logDescription)\(attributesDescription(from: attributes))"
+        let isSafe = attributes?["public"] as? Bool == true
+        guard isDebug || isSafe else {
+            // skips logs in production builds with non redacted info
+            return
+        }
```

Comment:
this restores the behavior of old logs, we would have less logs from users, it also means that when adding .safePublic the log should be safe of any user related info